Python/Anime-tracker/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Anime tracker  in python
+python script which will give info about searched anime such as :

```

Comment:
Replace `python` with `Python`. Reword the sentence to: `Python Script which will fetch information about a particular anime. The information fetched includes:`